README.md

```diff
@@ -69,27 +71,225 @@ class Genre {
+becomes _dependent_ on the second.
+
+In our `Book` `class`, for instance, we've written `infoStatement` using
+`this.author.fullname`. The `infoStatement` getter is expecting there to be a
+`fullname` property (or pseudo-property) present in `this.author`. If we
+mistakenly created a `Book` instance using only `String`s again and then tried
+to use `infoStatement` we will have a problem:
+
+```js
+let unfinishedTales = new Book(
+	'Unfinished Tales',
+	'J.R.R. Tolkien',
+	'Fantasy',
+	'1980'
+);
+
+unfinishedTales;
+// => Book {
+//    title: 'Unfinished Tales',
+//    author: 'J.R.R. Tolkien',
+//    genre: 'Fantasy',
+//    publishingDate: '1954' }
+
+unfinishedTales.author;
+// => 'J.R.R. Tolkien'
+unfinishedTales.author.fullname;
+// => undefined
+unfinishedTales.infoStatement;
+// => 'Unfinished Tales was written by undefined and published in 1980.'
+```
+
+Because of what we have written, `Book` is has become dependent on `Author` in
+order to fully function correctly. If we make changes to the `Author` `class`,
+say we change the name of a property, we could potentially break functionality
+in `Book`.
+
+On the flip side, if we change a property in `Book`, `Author` isn't affected.
+`Author` is not dependent on `Book`.
+
+Now, we could _choose_ to reverse this relationship and have `Author` dependent
+on `Book`. That might look something like the following:
+
+```js
+// notice that book is no longer keeping track of author
+class Book {
+	constructor(title, genre, publishingDate) {
+		this.title = title;
+		this.genre = genre;
+		this.publishingDate = publishingDate;
+	}
+}
+
+// a new Author instance will be given a _books property
+class Author {
+	constructor(firstName, lastName) {
+		this.firstName = firstName;
+		this.lastName = lastName;
+		this._books = [];
+	}
+
+	set books(books) {
+		this._books = books;
+	}
+
+	get books() {
+		return this._books;
+	}
+
+	get bookTitles() {
+		return this._books.map(book => book.title);
+	}
+}
+
+let stephenKing = new Author('Stephen', 'King');
+let it = new Book('It', 'Horror', '1986');
+let theShining = new Book('The Shining', 'Horror', '1977');
+
+stephenKing.books = [...stephenKing.books, it];
+stephenKing.books = [...stephenKing.books, theShining];
+
+stephenKing;
+// => Author {
+//     firstName: 'Stephen',
+//     lastName: 'King',
+//     _books: [
+// 		    Book { title: 'It', genre: 'Horror', publishingDate: '1986' },
+// 		    Book { title: 'The Shining', genre: 'Horror', publishingDate: '1977' }
+// 	   ]
+//   }
+
+stephenKing.bookTitles;
+// => ['It', 'The Shining']
+```
 
-## SWBAT 2
+In the example above, the `Author` `class` now keeps track of `Book` instances.
+`Book`s have no idea who their `Author`s are. We've reversed the relationship,
+and `Author` is now dependent on `Book`.
 
 ## Conclusion
 
-## Resources
+When building multi `class` applications, deciding which `class`es have
+dependencies is ultimately based on the needs and purpose of your application.
```

Comment:
Add a 1 or 2 sentences that summarize the first 2 learning goals as well. Then go into the phrase "deciding which `class`es have dependencies is ultimately ..."